lib/plugin.zsh

```diff
@@ -4,7 +4,7 @@ typeset -g GEOMETRY_PLUGIN_SEPARATOR=${GEOMETRY_PLUGIN_SEPARATOR:-" "}
 # Define default plugins
 typeset -ga GEOMETRY_PROMPT_PLUGINS
 if [[ $#GEOMETRY_PROMPT_PLUGINS -eq 0 ]]; then
-  GEOMETRY_PROMPT_PLUGINS=(exec_time git)
+  GEOMETRY_PROMPT_PLUGINS=(exec_time git hg)
```

Comment:
Yeah, my idea was to have both working transparently for the user, so if they move between git and mercurial repositories the default prompt will have the same look and feel.

I basically mimicked git plugin that's why I have put much into mercurial plugin. In this case yeah, it makes sense to have a check.